api/src/legacy_soap_api/legacy_soap_api_blueprint.py

```diff
@@ -0,0 +1,8 @@
+from apiflask import APIBlueprint
+
+legacy_soap_api_blueprint = APIBlueprint(
+    "legacy_soap_api_blueprint",
+    __name__,
+    tag="Legacy SOAP API",
+    cli_group="legacy_soap_api",
+)
```

Comment:
Unless having some vague swagger docs for the endpoint is critical, I might suggest not using APIFlask blueprints/a lot of the APIFlask stuff for setting this up, and instead use stuff from Flask itself (it should all connect up the same). APIFlask specifically is trying to make REST endpoints and you might run into problems trying to make that work with a SOAP endpoint.